### ISSUE TITLE
Visually improve 'last 28 days' charts

### ISSUE THREAD

**Issue by tmcw** [internetarchive/openlibrary#2380]
### Is your feature request related to a problem? Please describe.

I'd like the charts for the 'last 28 days' metrics to be clean and crisp.

### Describe the solution you'd like

The current look is

![image](https://user-images.githubusercontent.com/32314/64898334-9c95fa80-d63b-11e9-8ad1-dfbc856183f0.png)

I don't propose any significant visual overhaul: only to

- [ ] Render the chart canvas element at 2x so that it's not blurry on retina screens
- [x] Fix the `<a>` link styling so that there isn't a little tail of underline extending to the left of the chart.

### Proposal & Constraints

- Rendering the chart at 2x has a tiny effect on performance. Rendering it at either 1x or 2x adds complexity.
- I'm not sure yet how the charts are rendered, so I'm not sure whether this is simple to implement or not.

**Comment by cdrini** [comment]
:+1: This sounds good to me! I think rendering at 2x all the time is ok. The line should be easy to remove. @jdlrobson 's been working on a big refactor of this code in progress (#2360). @jdlrobson how hard do you think upscaling will be? Should this wait on that PR or can it go ahead?

**Comment by jdlrobson** [comment]
I think waiting on #2360 would be better.

**Comment by cdrini** [comment]
(If you're interested in other HTML/JS changes, these might be down your alley (in ~order of difficulty):
- #2373 : small fix to cutoff iframe
- #2132 : show visual indicator when changing cover
- #2144 : use `srcset` to avoid covers being blurry on highres displays )

I'll take a look at that PR shortly to help move it along as well :+1:

**Comment by cdrini** [comment]
#2360 has been merged :+1:

**Comment by tfmorris** [comment]
You'll need to force refresh your browser to get these graphs to render again after the new code update.

**Comment by guyjeangilles** [comment]
Is there anything else blocking this now that #2360 is merged?

**Comment by jdlrobson** [comment]
No this can be worked on.

**Comment by tmcw** [comment]
Yep, I'm still planning to do it, it's just taking a little while to get Docker set up again.

**Comment by guyjeangilles** [comment]
@tmcw Is there anything specifically holding you back with Docker?

**Comment by mekarpeles** [comment]
hey @tmcw, is this still an issue of interest?

**Comment by Yashs911** [comment]
@mekarpeles I would like to work on this issue. I need some guidance regarding how to Render the chart canvas element at 2x?

**Comment by Atheriz-46** [comment]
Hey, I would like to work on this issue if it's available. @tmcw

**Comment by tmcw** [comment]
Anyone's welcome to work on this issue if they want to and the folks reviewing PRs don't see any potential conflicts.

**Comment by RayBB** [comment]
@tmcw I would like to work on this.
I think I'll make a small PR to address the `<a>` underline by the chart first.

I can make a separate ticket, but I also think it would be nice to center align the charts to match the text below.
Here you can see the top row is what it looks like with chart centered. Below that is what it currently looks like with charts left aligned.
![image](https://user-images.githubusercontent.com/921217/112330089-d47ff400-8c5b-11eb-93e4-ba71ad47d49e.png)

**Comment by ma8642** [comment]
Just created a PR to address part of this issue:  the little tail at the bottom of the charts.

**Comment by jonbell-lot23** [comment]
Hello! I just checked these graphs on my phone and iPad and they look pretty crisp, though I haven’t looked into the code. Is this still an issue?

**Comment by tfmorris** [comment]
If I zoom in to the scale shown in the screen grab above, I still see it, but at normal resolution, I don't think it's a major issue for non-eagle-eyed. Ideally, if feasible, perhaps an SVG graph instead of a rendered image would be the best solution.

**Comment by mekarpeles** [comment]
@jonbell-lot23 here's a list of good first issues which we've identified as "help wanted"!
https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22+label%3A%22Needs%3A+Help%22

For this specific issue, you're welcome to tackle it, though you may find others in the list above which have higher impact potential. Let me know if there's anything else I can do to help!

**Comment by mekarpeles** [comment]
I think the quality seems sufficient now unless there are still complaints
